### DISCUSSION TITLE
serverless functions in go are not working

### DISCUSSION THREAD

**Discussion by blanc42** [vercel/community#2139] [Help] [answered]
### Summary

I tried to use serverless functions with golang but the .go in the api directory are being ignored

here is a part of build logs

Generated build outputs:
--
16:27:07.626 | - Static files: 25
16:27:07.626 | - Prerenders: 0
16:27:07.626 | - Serverless Functions: 0 <<<-- no functions
16:27:07.626 | - Edge Functions: 0
16:27:07.626 | Deployed outputs in 1s

### Additional information

// pages/api/hello.go

package handler

import (
	"fmt"
	"net/http"
)

func Handler(w http.ResponseWriter, r *http.Request) {
	fmt.Fprintf(w, "<h1>Hello from Go!</h1>")
}

---
- I tried serverless functions for go,python,ts,js
- but everything except ts and js is being ignored

**Comment by amyegan** [comment, marked as answer]
Hi @blank1729! Can you tell me more about your app? If you are using Next.js, only Node should go in `/pages/api`. Other runtimes belong in `/api` for Vercel to recognize. You can get more info in the docs for [Serverless Function Runtimes](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes#) and [Using the Go Runtime with Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/go#)

**Reply by blanc42** [reply]
Thanks for the reply. 
I was using src/api before. I moved the api folder to root directory and now it's working
